packages/dataclass-editor/src/master/ValidationRow.test.ts

```diff
@@ -1,28 +1,28 @@
-import { customRenderHook } from '../context/test-utils/test-utils';
-import type { DataClass } from '../data/dataclass';
-import { rowClassName, useUpdateOrder } from './ValidationRow';
+// import { customRenderHook } from '../context/test-utils/test-utils';
+// import type { DataClass } from '../data/dataclass';
+import { rowClassName } from './ValidationRow';
 
-test('useUpdateOrder', () => {
-  const dataClass = {
-    fields: [{ name: 'field0' }, { name: 'field1' }, { name: 'field2' }, { name: 'field3' }, { name: 'field4' }]
-  } as DataClass;
-  let newDataClass = {} as DataClass;
-  const view = customRenderHook(() => useUpdateOrder(), {
-    wrapperProps: { appContext: { dataClass, setDataClass: dataClass => (newDataClass = dataClass) } }
-  });
+// test('useUpdateOrder', () => {
```

Comment:
Since I removed the useUpdateOrder hook, this test no longer works. Testing the updateOrder function has become more challenging, as I now need a react table to retrieve the selected rows to use the updateOrder function properly. I am not sure how to solve this:
- I could recreate the useUpdateOrder hook, place the table in the AppContext, and test everything in isolation (as before).
- Alternatively, I could create a unit test with a mock table.
- Another option is to write an integration test instead.

Which approach would be best?

Reply:
Fine for me if this hook no longer makes sense.

Just move the updateOrder function to it's own file, then you can test it.

Reply:
Alright, i will have a look at this updateOrder-test in a later PR 👍

Reply:
yes ok for me